proto/frequenz/api/microgrid/microgrid.proto

```diff
@@ -540,37 +610,32 @@ message SetPowerReactiveParam {
   float power = 2;
 }
 
-// Parameters for setting bounds of a given metric of a given component.
-message SetBoundsParam {
-  // An enumerated list of metrics whose bounds can be set.
-  enum TargetMetric {
-    TARGET_METRIC_UNSPECIFIED = 0;
-    TARGET_METRIC_POWER_ACTIVE = 1;
-    TARGET_METRIC_CURRENT = 2;
-    TARGET_METRIC_CURRENT_PHASE_1 = 3;
-    TARGET_METRIC_CURRENT_PHASE_2 = 4;
-    TARGET_METRIC_CURRENT_PHASE_3 = 5;
-    TARGET_METRIC_POWER_REACTIVE = 6;
-  }
-
-  // The ID of the target component.
+// Request parameters for the RPC `StartComponent`.
+message StartComponentRequest {
+  // The component ID to start.
   uint64 component_id = 1;
+}
 
-  // The target metric whose bounds have to be set.
-  TargetMetric target_metric = 2;
+// Request parameters for the RPC `HotStandbyComponent`.
+message HotStandbyComponentRequest {
+  // The component ID to.
```

Comment:
Nitpick: The component ID to what?

Here and below :)

Reply:
oops :see_no_evil: 
Fixed!